src/client/gameui.cpp

```diff
@@ -63,9 +63,18 @@ void GameUI::init()
 
 	// At the middle of the screen
 	// Object infos are shown in this
+	unsigned int chat_height;
+	u16 chat_font_size = g_settings->getU16("chat_font_size");
+	if (chat_font_size != 0) {
+		chat_height = g_fontengine->getFont(chat_font_size, FM_Unspecified)->getDimension(L"").Height;
```

Comment:
Usually the font height is checked using `getDimension("Ay")` to take lower parts into account as well.